js/tokens.ts

```diff
@@ -174,8 +187,20 @@ export type TSelectedSet = Set<string | number>;
 
 export const emptyArray = Object.freeze([]);
 
+export interface IDynamicCallable {
+  (...args: any): string;
+}
+
 export type TCustomAction = 'reheat';
 
 export interface IActionMessage {
   action: TCustomAction;
 }
+
+export const FALSEY = ['', '0', 'false'];
```

Comment:
should we add `null`, `[]`, and/or `{}`?

Reply:
5cdca93